README.md

```diff
@@ -11,62 +11,75 @@ The hazen-web-app **requires** manual installation of the following software:
 - hazen-web-app (i.e.: this repo) - requires Python3.8 or above and `pip`
 
 
-## Setup using docker compose
-To start the application:
-1. Create .env file and read variables. Can reuse default.env for testing or recommend creating your own in production.
-```
-mv default.env .env
-source .env
-```
-
-2. Start web app
-```
-docker compose up -d --build
-```
+## Table of Contents
+- Setting up for the first time
+  - [on MacOs](#setup-on-macos)
+  - [on Linux](#setup-on-linux-ubuntu)
+- Running locally
+  - [how to make and test code changes](#how-to-make-and-test-code-changes)
+  - [using docker compose](#setup-using-docker-compose)
+- Contributing to development and documentation
+  - [process for contributing](#process-for-contributing)
+  - [update documentation](#update-documentation)
+  - Release Process - to be devised
 
-The hazen web app is then accessible on port 8080
 
 ## Setup on MacOS
 
-### Postgres & Postico
+### Database - Postgres & Postico
 
-Official Postgres.app installation instructions: [Postgres.app](https://postgresapp.com/). Can also be installed using Homebrew.
+1. Installation via Homebrew is recommended, instructions at [Postgres wiki](https://wiki.postgresql.org/wiki/Homebrew),
+or see the official [Postgres.app](https://postgresapp.com/) installation instructions.
 
 In Terminal:
 ```shell
 brew update
+# Install postgresql
 brew install postgresql
+brew services start postgresql
+
+# you can also stop and restart the postgres service
+brew services stop postgresql
+brew services restart postgresql
 ```
-*Confirm PostgreSQL is running before moving onto the next step.* You can do this by opening the Postgres.app, 
-or if you prefer to use Terminal you can start it following the instructions [here](https://sourabhbajaj.com/mac-setup/PostgreSQL/).
 
-Download [Postico](https://eggerapps.at/postico/).
+2. Create a new database with the following details:
+*Confirm PostgreSQL is running before moving onto the next step.*
+You can do this by opening the Postgres.app, or if you prefer to use Terminal you can start it following the instructions [here](https://sourabhbajaj.com/mac-setup/PostgreSQL/).
```

Comment:
might be worth saying pgAdmin instead of Postgres.app (not sure if Postgres.app is specific for Mac tho)